nautobot/ipam/api/views.py

```diff
@@ -391,6 +391,51 @@ class VLANGroupViewSet(NautobotModelViewSet):
     serializer_class = serializers.VLANGroupSerializer
     filterset_class = filters.VLANGroupFilterSet
 
+    @action(
+        detail=True,
+        name="Available VLAN IDs",
+        url_path="available-vlans",
+        methods=["get"],
+        filterset_class=None,
+    )
+    def available_vlans(self, request, pk=None):
```

Comment:
At some point will need to `extend_schema` here to get the correct representation in OpenAPI/Swagger. I'm fine with that coming in a future PR but let's make sure not to forget about it.

Reply:
Thanks, yes indeed, this will be added